src/main/java/org/apache/flink/checkpoint/benchmark/FsStateBackendRescalingBenchmarkExecutor.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.checkpoint.benchmark;
```

Comment:
actually I have one more question. Why is this in `checkpoint` package? Why not the pre-existing `org.apache.flink.state.benchmark`?

Note that adding `checkpoint` it would complicate the jenkins/codespeed setup and in offline discussion with @Myasuka  we agreed to not do it (to run those rescaling benchmarks together with other state backend benchmarks). To do so it would be I think easiest to add the rescaling benchmarks to the `org.apache.flink.state.benchmark` package. Otherwise we would need to modify `pom.xml`/jenkins exclusion/inclusion patterns for selecting the benchmarks.

Reply:
👍fixed, moved the rescaling benchmarks to the `org.apache.flink.state.benchmark` package.